### DISCUSSION TITLE
Allow to disable fetch logs

### DISCUSSION THREAD

**Discussion by eps1lon** [vercel/next.js#50833] [Ideas]
### Goals

1. Debugging server issues unrelated to `fetch` or other Next.js functionality

### Background

Trying to figure out a server crash on a page with plenty of fetch requests. Fetch logs take up half of my screen making it harder to scan for the relevant logs.

### Proposal

Add option to configure log level. Fetch logs are info or verbose and should be omitted if I set the log level to e.g. `warn`. Alternatively, allow disabling fetch logs entirely (temporarily) instead of a global log level.

**Comment by dancherb** [comment]
Agreed. Did you find a way to achieve this?

**Comment by NickG-NZ** [comment]
+1 to this. We are big boys (and girls), we set up our own logging. We don't want to see the Vercel logs.

**Comment by CaptainN** [comment]
Some way to toggle these during development would be a god send...

**Comment by Heryk13** [comment]
Yep i want this too
